include/picongpu/fields/MaxwellSolver/YeePML/Field.tpp

```diff
@@ -92,9 +65,10 @@ namespace yeePML
         return data->getDeviceBuffer( ).getDataBox( );
     }
 
-    GridBuffer< Field::ValueType, simDim > & Field::getGridBuffer( )
+    EventTask Field::asyncCommunication( EventTask serialEvent )
     {
-        return *data;
+        EventTask eB = data->asyncCommunication( serialEvent );
```

Comment:
suggestion: `return data->asyncCommunication( serialEvent );` since the name `eB` has for me no obvious meaning.